src/resume_parser/app2.py

```diff
@@ -0,0 +1,69 @@
+from flask import Flask, render_template, request, jsonify
+import os
+import re
+import nltk
+from werkzeug.utils import secure_filename
+from docx import Document
+from PyPDF2 import PdfReader
+from nltk.corpus import stopwords
+from collections import Counter
+
+nltk.download('stopwords')
+
+app = Flask(__name__)
+app.config['UPLOAD_FOLDER'] = './uploads'
+
+
+os.makedirs(app.config['UPLOAD_FOLDER'], exist_ok=True)
+
+def extract_text_from_docx(file_path):
+    doc = Document(file_path)
+    return "\n".join([paragraph.text for paragraph in doc.paragraphs])
+
+def extract_text_from_pdf(file_path):
+    text = ""
+    with open(file_path, 'rb') as file:
+        reader = PdfReader(file)
+        for page in reader.pages:
+            text += page.extract_text()
+    return text
+
+def extract_keywords(text):
```

Comment:
Could you elaborate on what criteria are we parsing keywords  ? This seems to be parsing all the words in the english dictinoary expect grammar words. Do you think there is a way we can make our parser more streamlined ?

Reply:
Resolved in the next pull request